pkg/utils/utils.go

```diff
@@ -244,6 +244,24 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 				if iface.Mtu > 0 {
 					mtu = iface.Mtu
 				}
+
+				// validate the device exist only with the default driver
+				// TODO: remove this work around after the BZ is close
+				// https://bugzilla.redhat.com/show_bug.cgi?id=1875338
+				b := backoff.NewConstantBackOff(1 * time.Second)
+				err = backoff.Retry(func() error {
+					_, err := dputils.GetNetNames(addr)
+					if err != nil {
+						glog.Warningf("configSriovDevice():: fail to get interface name for %s: %s", addr, err)
+						return err
+					}
+					return nil
+				}, backoff.WithMaxRetries(b, 10))
+				if err != nil {
+					glog.Warningf("configSriovDevice(): unbind PCI %s from default driver and return the error: %v", addr, err)
+					return Unbind(addr)
```

Comment:
Not sure I am missing something, but shouldn't we return an error here? If we just unbind, the function will exit successfully

Reply:
you are right I miss that!